### ISSUE TITLE
Decode script does not generate 1Best.ctm

### ISSUE THREAD

**Issue by akashrajkn** [opensource-spraakherkenning-nl/Kaldi_NL#12]
Hi, thank you for open sourcing this project.

I am trying to run the `nnet3_online tdnn` model. On running the `decode.sh` script, I get the following output

```
Argument filename.wav is a sound file, using it as audio
Diarization completed in 0:00:47 (CPU: 0:00:40), Memory used: 117 MB                
Split 1 source file into 103 segments                              
Duration of speech: 0h:10m:23s
NNet3 decoding completed in 0:00:00 (CPU: 0:00:00), Memory used: 4 MB                
Rescoring completed in 0:00:00 (CPU: 0:00:00), Memory used: 0 MB                
Rescoring completed in 0:00:00 (CPU: 0:00:00), Memory used: 2 MB                
Done
```

However <output-dir> does not contain `1Best.ctm` results file. Any help is appreciated.

**Comment by akashrajkn** [comment]
On investigation of `output/intermediate/log` file, I see that the following files are missing:

```
models/NL/UTwente/HMI/AM/CGN_all/nnet3_online/tdnn/v1.0/graph_UTwente_HMI_KrantenTT_v1.0_KrantenTT.3gpr.kn.int_UTwente_HMI_lexicon/HCLG.fst

models/NL/UTwente/HMI/LM/KrantenTT/v1.0/LG_KrantenTT.3gpr.kn.int_UTwente_HMI_lexicon/G.fst
```

**Comment by thatux** [comment]
same problem here, I'm using WSL, everything seems to be fine, but no Best1.cms

command

`
./decode_PR.sh /mnt/e/filtered_short.mp3 output/3
Argument /mnt/e/filtered_short.mp3 is a sound file, using it as audio
[                                                  ] Diarization (0/1)
Diarization completed in 0:01:13 (CPU: 0:01:34), Memory used: 589 MB
Split 1 source file into 361 segments
Duration of speech: 0h:50m:0s
NNet3 decoding completed in 0:00:00 (CPU: 0:00:01), Memory used: 5 MB
Rescoring completed in 0:00:00 (CPU: 0:00:00), Memory used: 0 MB
Rescoring completed in 0:00:00 (CPU: 0:00:00), Memory used: 1 MB
Done
`

log
`
./decode_PR.sh /mnt/e/filtered_short.mp3 output/3
cat: 'output/3/intermediate/data/*.stm': No such file or directory
fix_data_dir.sh: kept all 361 utterances.
fix_data_dir.sh: old files are kept in output/3/intermediate/data/ALL/.backup
steps/make_mfcc.sh --nj 8 --mfcc-config output/3/intermediate/mfcc.conf output/3/intermediate/data/ALL output/3/intermediate/data/ALL/log output/3/intermediate/mfcc
Usage: utils/validate_data_dir.sh [--no-feats] [--no-text] [--non-print] [--no-wav] [--no-spk-sort] <data-dir>
The --no-xxx options mean that the script does not require 
xxx.scp to be present, but it will check it if it is present.
--no-spk-sort means that the script does not require the utt2spk to be 
sorted by the speaker-id in addition to being sorted by utterance-id.
--non-print ignore the presence of non-printable characters.
By default, utt2spk is expected to be sorted by both, which can be 
achieved by making the speaker-id prefixes of the utterance-ids
e.g.: utils/validate_data_dir.sh data/train
steps/compute_cmvn_stats.sh output/3/intermediate/data/ALL output/3/intermediate/data/ALL/log output/3/intermediate/mfcc
steps/compute_cmvn_stats.sh: no such file output/3/intermediate/data/ALL/feats.scp
steps/online/nnet3/decode.sh --nj 8 --acwt 1.2 --post-decode-acwt 10.0 --skip-scoring true models/AM/graph_PR output/3/intermediate/data/ALL output/3/tmp/tmp.Xzb6VJ5zsx
filter_scps.pl: warning: some input lines were output to multiple files [OK if splitting per utt] 
run.pl: 8 / 8 failed, log is in output/3/tmp/tmp.Xzb6VJ5zsx/log/decode.*.log
steps/lmrescore_const_arpa.sh --skip-scoring true models/Lang_PR models/LM/Const_PR output/3/intermediate/data/ALL output/3/intermediate/decode output/3/intermediate/rescore
ls: cannot access 'output/3/intermediate/decode/lat.*.gz': No such file or directory
steps/lmrescore_const_arpa.sh: No lattices input directory output/3/intermediate/decode
`

**Comment by proycon** [comment]
Sorry for the long delay! I'm investigating these issues now.

**Comment by proycon** [comment]
I have just released Kaldi-NL v0.3 which should hopefully fix this problem. Please let me know if it still persists.
